README.md

```diff
@@ -9,31 +9,15 @@ interpreter.  It was originally based on
 [vim-slime](https://github.com/jpalardy/vim-slime).
 
 **New**: If you are using NeoVim, you can now use the Terminal instead of tmux.
-Read the Configuration section on how to enable it.
+Read the Configuration section on how to enable it. Likewise, if you are using
+Vim8, this extensions uses the terminal feature.
 
-![](http://i.imgur.com/frOLFFI.gif)
+[![asciicast](https://asciinema.org/a/224891.svg)](https://asciinema.org/a/224891)
 
 ## Getting Started ##
 
-1. Start livecoding with Vim by simply running:
-
-   ```bash
-   $ tidalvim
-   ```
-
-   This creates a tmux session with Vim and Tidal running on different panes.
-
-2. Write something like this:
-
-   ```haskell
-   d1 $ sound "bd sn"
-   ```
-
-3. While being on that line, press `<c-e>` (Control + E) to evaluate it.
-
-   You should see Vim flash that line for a second and a chunk of text appear on
-   your Tidal interpreter.  If you already have SuperDirt or other synth running,
-   you should hear a kick and a snare :)
+   When inside Vim, run the command `:TidalStart`.
```

Comment:
It doesn't seem like there's a `:TidalStart` command in this branch? Hitting CTRL+E on a line of code starts up Tidal just fine though.

Reply:
Good catch! I had added TidalStart when I was working on VIM8 support on its own since I thought it could be useful to start the Tidal interpreter without having to send something first. Maybe we could make TidalStart availble again?

Reply:
+1 to add `:TidalStart`. Likewise, there should be a `:TidalStop` and `:TidalRestart` I believe. It's useful to restart GHCI sometimes as Tidal might crash and the only way to recover is to actually close GHCI and reload the Tidal modules. But these could be added in another issue.